src/test/messenger/CMakeLists.txt

```diff
@@ -1,43 +0,0 @@
-add_executable(simple_server
```

Comment:
we need to remove `add_subdirectory(messenger)` in `src/test/CMakeLists.txt`, just pushed the updated change to this PR.